cmd/scollector/collectors/program.go

```diff
@@ -151,12 +178,20 @@ Loop:
 			Value:     val,
 			Tags:      opentsdb.TagSet{"host": util.Hostname},
 		}
-		for _, tag := range sp[3:] {
-			tags, err := opentsdb.ParseTags(tag)
+		for _, field := range sp[3:] {
+			if strings.HasPrefix(field, "meta.") {
```

Comment:
This is an interesting solution. @kylebrandt thoughts? The other kind of solution to consider is just a line completely devoted to metadata. This would allow the metadata to be printed, say, just once on collector startup instead of being forced to output with every or just the first point, which is annoying for external collector authors. It would also allow the meta namespace to be used as is.